tools/ci/src/main.rs

```diff
@@ -220,16 +220,14 @@ cargo run {github_token_auth_flag}{target} -p spacetimedb-update -- self-install
                 );
             }
 
-            bash!("pnpm install --recursive")?;
-            bash!("cargo run --features markdown-docs -p spacetimedb-cli > docs/docs/cli-reference.md")?;
-            bash!("pnpm format")?;
+            bash!("cd docs && pnpm generate-cli-docs")?;
             bash!("git status")?;
             bash!(
                 r#"
-if git diff --exit-code HEAD; then
+if [ -z "$(git status --porcelain)" ]; then
```

Comment:
this should also error on new files, which was how this silently gotbroken before